widgets/contact/contact.php

```diff
@@ -730,7 +730,7 @@ function get_template_variables( $instance, $args ) {
 		$instance_hash = md5( serialize( $instance ) );
 		unset( $instance['_sow_form_id'] );
 
-		if ( ! empty( $instance['settings']['submit_attributes']['submit_id'] ) ) {
+		if ( ! empty( $instance['settings']['submit_id'] ) ) {
```

Comment:
This was defined as `id` in the `settings` form field, so either need to change this to `$instance['settings']['id']` or change the form field key to `submit_id`

Reply:
Whoops - sorry.